README.md

```diff
@@ -95,10 +95,15 @@ deploying and running an example chaincode.
 ## Requirements
 
 * CMake 3.5.1 or higher
-* Go 1.11 or higher
-* Install Fabric v1.4 https://github.com/hyperledger/fabric 
-* Install the Linux SGX SDK v2.4 https://github.com/intel/linux-sgx 
-* You also need to install Intel SGX SSL https://github.com/intel/intel-sgx-ssl
+* Go 1.11.x or higher
+* Hyperledger Fabric v1.4 https://github.com/hyperledger/fabric 
+
+    If you are new to Fabric, we recommend the Fabric documentation as your starting point. You should start with
+    [installing](https://hyperledger-fabric.readthedocs.io/en/release-1.4/prereqs.html) Fabric dependencies and setting
+    up your [development environment](https://hyperledger-fabric.readthedocs.io/en/release-1.4/dev-setup/build.html).
+    
+* SGX SDK v2.4 for Linux https://github.com/intel/linux-sgx 
+* SSL for SGX SDK v2.4 https://github.com/intel/intel-sgx-ssl
```

Comment:
maybe we should switch sgxssl to v2.4.1? And as discussed during call propably mention to use latest version of openssl 1.1.0 (greater-or-equal 1.1.0j)?

Reply:
Done!